coalals/interface.py

```diff
@@ -0,0 +1,60 @@
+import sys
+from os import chdir
+from json import dumps
+from io import StringIO
+from coalib import coala
+from contextlib import redirect_stdout
+from concurrent.futures import ProcessPoolExecutor
+from .utils.log import configure_logger, reset_logger
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class coala_wrapper:
```

Comment:
About the class name, I think we have two patterns: coala_wrapper and LangServer. Could we unify the way?